examples/mnist/Instructions.md

```diff
@@ -0,0 +1,25 @@
+<p align="center"><b>How to test example MNIST train/test programs</b><p align="center">
+
+
+
+<b>Step:1</b> Install Cmake from https://cmake.org/install/  
+<b>Step:2</b> Open Terminal.  
+<b>Step:3</b>
+```bash
+cd ~/install_path # adjust `install_path` where you want to install tiny-dnn
```

Comment:
I think `to` is missing in the above sentence.
cd install_path # adjust `install_path` to where you want to clone tiny-dnn git repository